lib/improver/utilities/spatial.py

```diff
@@ -240,6 +266,11 @@ def process(self, cube):
             cube (Iris.cube.Cube):
                 Cube from which the differences will be calculated.
 
+        Kwargs:
+            gradient (boolean):
+                Optionally return gradient rather than difference.  This has
+                dimensions of original grid, rather than losing 1 row & col.
```

Comment:
'...This is achieved by linearly interpolating the gradients to the grid of the input cube.'

Or something to that effect.